modules/aws_ecs/locals.tf

```diff
@@ -71,15 +73,15 @@ locals {
       # Workflows-specific
       {
         "name" : "WORKFLOW_BACKEND_HOST",
-        "value" : "http://workflow-backend.retoolsvc:3000"
+        "value" : format("http://workflow-backend.%s:3000", local.service_discovery_namespace)
       },
       {
         "name" : "WORKFLOW_TEMPORAL_CLUSTER_NAMESPACE",
         "value" : var.temporal_cluster_config.namespace
       },
       {
         "name" : "WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_HOST",
-        "value" : var.temporal_cluster_config.host
+        "value" : format("%s.%s", var.temporal_cluster_config.hostname, local.service_discovery_namespace)
```

Comment:
Why did `host` become `hostname`?

Reply:
Because Terraform only supports constants in defaults, and the original default value of `temporal.retoolsvc` doesn't make sense anymore with the DNS name being dynamic.